cmd/samples/http/receiver/main.go

```diff
@@ -41,7 +40,7 @@ func gotEvent(ctx context.Context, event cloudevents.Event) error {
 	}
 	fmt.Printf("Got Data: %+v\n", data)
 
-	fmt.Printf("Got Transport Context: %+v\n", http.TransportContextFrom(ctx))
+	fmt.Printf("Got Transport Context: %+v\n", cloudeventshttp.TransportContextFrom(ctx))
```

Comment:
other samples are using cehttp...do we want consistency in this naming?

Reply:
I have not made a choice on which is best.